### PR TITLE
Add rudimetry .tsx support

### PR DESCRIPTION
**Pull request by tmhmitchell**
Basic support for loading external (ie, in seperate .tsx files) Tilesets.

N.B., this doesn't implement the entire Tileset spec - my use case isn't particularly complicated, so I've just added what I need for now.

### CODE REVIEW

**Comment by Rulox** [comment]
Hey @tmhmitchell thanks for this! I'll check and merge this soon

File: ebitsx.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	"io/ioutil"
+	"math"
+	"os"
+)
+
+// EbitenTileset is a friendly representation of a TSX Tileset
+type EbitenTileset struct {
+	TileWidth  int
+	TileHeight int
+	SetWidth   int
```
Comment (Rulox) [inline ebitsx.go:15]:
```suggestion
	TilesetWidth   int
```

File: ebitsx.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	SetHeight  int
```
Comment (Rulox) [inline ebitsx.go:16]:
```suggestion
	TilesetHeight  int
```

**Review by Rulox — APPROVED** [review]
It looks great thanks!

I suggested 2 name changes, the reason is that I would like to make certain properties private in the future and access them with 'GetTilesetWidth' for instance, I feel if we have SetWidth, it might be confusing as 'Set' is often using as a verb (for Setters).

What do you think? Apart from that looks great! Can't wait to merge it

PS: Do you have any small example that we can add to the /examples folder? If not is fine, I'll create one

**Comment by tmhmitchell** [comment]
Sound good to me - I'll make those changes and put together an example

**Comment by Rulox** [comment]
Thanks @tmhmitchell ! :)

**Pull request merged**